server/routes/api/moderator/moderatorHandler.ts

```diff
@@ -1,5 +1,12 @@
 import DBHandler from "./moderatorDBHandler";
-import { User, ContactRequest } from "./moderatorTypes";
+import { User, ContactRequest, Validation } from "./moderatorTypes";
+
+enum RequestStatuses {
```

Comment:
why did you create an enum and not an array?

Reply:
because the elements are constants